### PR TITLE
Add warnings to series

### PR DESCRIPTION
**Pull request by jacksontj**
Signed-off-by: Thomas Jackson <jacksontj.89@gmail.com>

cc @krasi-georgiev

### CODE REVIEW

**Comment by jacksontj** [comment]
FYI I've also opened https://github.com/prometheus/prometheus/issues/5661 as this warnings return seems like it would need to be included in `LabelValues` and `LabelNames` as well.

**Comment by krasi-georgiev** [comment]
Is this already in prometheus? can you put a link to the code.

**Comment by jacksontj** [comment]
Yes, this is already in prometheus (https://github.com/prometheus/prometheus/blob/master/web/api/v1/api.go#L492). I have opened up the other issue because logically this same `Warnings` should exist for those other 2 APIs as well (but don't today).

**Comment by krasi-georgiev** [comment]
can you add a test for this one as well?

**Comment by jacksontj** [comment]
@krasi-georgiev sure, I thought I had already -- but that was for the other PR, here we go :)

File: api/prometheus/v1/api_test.go [hunk 1]
```diff
@@ -376,6 +401,21 @@ func TestAPIs(t *testing.T) {
 			},
 			err: fmt.Errorf("some error"),
 		},
+		// series with error and warning
```
Comment (krasi-georgiev) [inline api/prometheus/v1/api_test.go]: capital letter and full stop for all comments :)

**Comment by jacksontj** [comment]
@krasi-georgiev that should do it.

**Comment by krasi-georgiev** [comment]
Thanks!

**Pull request merged**